.github/workflows/release-please.yml

```diff
@@ -35,55 +37,94 @@ jobs:
+    needs: release-please
     permissions:
-      actions: read
       id-token: write
       contents: write
-    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.10.0
-    with:
-      base64-subjects: "${{ needs.release-sdk-server.outputs.hashes }}"
-      upload-assets: true
-      upload-tag-name: ${{ needs.release-please.outputs.package-sdk-server-tag_name }}
-      provenance-name: ${{ format('LaunchDarkly.ServerSdk-{0}_provenance.intoto.jsonl', needs.release-please.outputs.package-sdk-server-tag_name) }}
+      pull-requests: write
+    if: ${{ needs.release-please.outputs.package-sdk-server-telemetry-released == 'true'}}
+    outputs:
+      hashes: ${{ steps.full-release.outputs.hashes }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-env
+        with:
+          workspace_path: LaunchDarkly.ServerSdk.Telemetry
 
-  release-telemetry:
-    runs-on: ubuntu-latest
+      - uses: ./.github/actions/full-release
+        id: full-release
+        with:
+          workspace_path: ${{ env.WORKSPACE_PATH }}
+          project_file: ${{ env.PROJECT_FILE }}
+          build_output_path: ${{ env.BUILD_OUTPUT_PATH }}
+          test_project_file: ${{ env.TEST_PROJECT_FILE }}
+          dll_name: ${{ env.BUILD_OUTPUT_DLL_NAME }}
+          dry_run: ${{ inputs.dry_run }}
+          aws_role: ${{ vars.AWS_ROLE_ARN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  release-sdk-client:
+    runs-on: macos-latest
     needs: release-please
     permissions:
       id-token: write
       contents: write
       pull-requests: write
-    if: ${{ needs.release-please.outputs.package-sdk-server-telemetry-released == 'true'}}
+    if: ${{ needs.release-please.outputs.package-sdk-client-released == 'true'}}
     outputs:
       hashes: ${{ steps.full-release.outputs.hashes }}
     steps:
       - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-env
+        with:
+          workspace_path: LaunchDarkly.ClientSdk
+
       - uses: ./.github/actions/full-release
         id: full-release
         with:
-          workspace_path: 'pkgs/telemetry'
-          project_file: 'pkgs/telemetry/src/LaunchDarkly.ServerSdk.Telemetry.csproj'
-          test_project_file: 'pkgs/telemetry/test/LaunchDarkly.ServerSdk.Telemetry.Tests.csproj'
-          build_output_path: 'pkgs/telemetry/src/bin/Release/'
-          dll_name: 'LaunchDarkly.ServerSdk.Telemetry.dll'
-          dry_run: false
+          workspace_path: ${{ env.WORKSPACE_PATH }}
+          project_file: ${{ env.PROJECT_FILE }}
+          build_output_path: ${{ env.BUILD_OUTPUT_PATH }}
+          test_project_file: ${{ env.TEST_PROJECT_FILE }}
+          dll_name: ${{ env.BUILD_OUTPUT_DLL_NAME }}
+          dry_run: ${{ inputs.dry_run }}
           aws_role: ${{ vars.AWS_ROLE_ARN }}
           token: ${{ secrets.GITHUB_TOKEN }}
 
+  # TODO: refactor provenance
+  release-sdk-server-provenance:
```

Comment:
For reviewers: Moved provenance tasks to the bottom for organization.  Made the diff weird.